src/python/include/utils/utilities.h

```diff
@@ -0,0 +1,262 @@
+/*    This file is part of the MMA Library - https://gitlab.inria.fr/dloiseau/multipers - which is released under MIT.
+ *    See file LICENSE for full license details.
+ *    Author(s):       David Loiseaux, Hannah Schreiber
+ *
+ *    Copyright (C) 2022 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef UTILITIES_H
+#define UTILITIES_H
+
+#include <vector>
+#include <limits>
+#include <omp.h>
```

Comment:
For now we have been using TBB for parallelism in Gudhi. Since we now use C++17, there are also some standard parallel algorithms. Mixing many different parallelism frameworks may cause problems at some point. OpenMP is badly supported by visual studio, so it probably shouldn't be our first choice.